jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -473,25 +461,17 @@ IGN_COMMON_NO_IGN_DEPENDENCIES="pkg-config     \\
                          uuid-dev"
 
 IGN_COMMON_DEPENDENCIES="${IGN_COMMON_NO_IGN_DEPENDENCIES} \\
-                     libignition-cmake-dev \\
-                     libignition-cmake1-dev \\
-                     libignition-math4-dev"
```

Comment:
I believe `ignition-common1` uses `libignition-cmake-dev` and `libignition-math4-dev` and is a dependency of `gazebo9`, so if we wanted to make a pull request at `ign-common1`, it may need these packages still listed

Reply:
Ahh shoot, we have to update `ign-cmake0`'s status in https://github.com/ignitionrobotics/docs/blob/master/tools/versions.md

Reply:
See https://github.com/ignitionrobotics/docs/pull/189